server/models/post.js

```diff
@@ -21,12 +21,15 @@ const postSchema = new Schema(
 			default: Date.now,
 			get: (timestamp) => dateFormat(timestamp),
 		},
-		postAuthor: {
+		username: {
```

Comment:
I changed it from author post to username for my clarity we can change it back if needed